mers/CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing

```

Comment:
Can we move this to root, and have this as the main contributors guide?

Reply:
+1

Reply:
@unrahul we have automated Internally the "path files placing" in MeRS, just moving `CONTRIBUTING.md` is no problem ;) but we need to fine tune / adjust automation to fit this.
Could we merge it as it is, and to follow up this task I can create an issue.
Beside this, I believe it is a good idea to standardize `CONTRIBUTING.md`  -> `root` place and notify other stack folks before doing changes on this kind of files.

Thoughts here @DnPlas @unrahul ?

Reply:
Agree, we can do it in a follow on patch.